src/components/AppProvider/README.md

```diff
@@ -391,6 +429,19 @@ ReactDOM.render(
 
 ---
 
+## Using translations
+
+When using Polaris, you are able to import translations from over 10 different langauges and consume them through the `i18n` prop.
```

Comment:
Let's either list out the available languages here, or omit the number entirely in favor of something like `all languages supported by the core Shopify product`

Reply:
I can see us forgetting to update the docs when new languages are added, I'm in favor of the latter.